libmscore/paste.cpp

```diff
@@ -324,9 +324,10 @@ bool Score::pasteStaff(XmlReader& e, Segment* dst, int dstStaff)
 
                               // be sure to paste the element in the destination track;
                               // setting track needs to be repeated, as it might have been overwritten by el->read()
-                              el->setTrack(e.track());
+                              // preserve *voice* from source, though
+                              el->setTrack(e.track() + el->voice());
```

Comment:
Is there a chance e.track() would already have a voice offset added in?  It would always the first track of the staff in my tests.  I suppose safer might be to convert e.track() to staffIdx and then back to track before adding the voice?  The *original* code here did in fact use staffIdx.  That was changed here:

https://github.com/musescore/MuseScore/commit/5ed2bc8be843beee43497e4160a7a9f855fcf025#diff-7d7e4132f003b8764c359cfabafdf6beL303